torchvision/csrc/models/vgg.cpp

```diff
@@ -0,0 +1,139 @@
+#include "vgg.h"
+
+#include <map>
+#include "modelsimpl.h"
+
+namespace vision
+{
+namespace models
+{
+torch::nn::Sequential makeLayers(const std::vector<int> &cfg,
+								 bool batch_norm = false)
+{
+	torch::nn::Sequential seq;
+	auto channels = 3;
+
+	for (const auto &V : cfg)
+	{
+		if (V <= -1)
+			seq->push_back(modelsimpl::MaxPool2D(2, 2));
+		else
+		{
+			seq->push_back(torch::nn::Conv2d(
+				torch::nn::Conv2dOptions(channels, V, 3).padding(1)));
+
+			if (batch_norm) seq->push_back(torch::nn::BatchNorm(V));
+			seq->push_back(modelsimpl::Relu(true));
+
+			channels = V;
+		}
+	}
+
+	return seq;
+}
+
+void VGGImpl::_initialize_weights()
+{
+    for (auto &module : modules(false))
+    {
+        if (torch::nn::Conv2dImpl *M =
+                dynamic_cast<torch::nn::Conv2dImpl *>(module.get()))
+        {
+            torch::nn::init::xavier_normal_(M->weight);
+            torch::nn::init::constant_(M->bias, 0);
+        }
+        else if (torch::nn::BatchNormImpl *M =
+					 dynamic_cast<torch::nn::BatchNormImpl *>(module.get()))
+        {
+            torch::nn::init::constant_(M->weight, 1);
+            torch::nn::init::constant_(M->bias, 0);
+        }
+        else if (torch::nn::LinearImpl *M =
+					 dynamic_cast<torch::nn::LinearImpl *>(module.get()))
+        {
+            torch::nn::init::normal_(M->weight, 0, 0.01);
+            torch::nn::init::normal_(M->bias, 0);
```

Comment:
I believe this should be `constant` with 0
https://github.com/pytorch/vision/blob/1f8f7ea71c8807408a7860b8ae9ca63177975b64/torchvision/models/vgg.py#L55-L57